### PR TITLE
Expand test coverage.

### PR DESCRIPTION
**Pull request by proofit404**

### CODE REVIEW

File: src/mappers/_mapper.py [hunk 1]
```diff
@@ -280,19 +283,10 @@ def validate_fields(fields, data_source, config):
...
-                            getattr(model_field, "attname", object()), str
-                        )
-                        model = model_field.related_model  # type: ignore
-                        break
-                else:
-                    raise Exception(
-                        "Can not find '%s' field in the %s model" % (field, model)
-                    )
+                model_field = get_data_source_field(path_field, model)
+                assert model_field.is_relation  # type: ignore
```
Comment (sobolevn) [inline src/mappers/_mapper.py]: It is better not to rely on `assert`, because it is striped from the source code when `-O` is used.

Comment (sobolevn) [reply src/mappers/_mapper.py]: Use `raise AssertionError()`

File: src/mappers/exceptions.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+:copyright: (c) 2019 by dry-python team.
+:license: BSD, see LICENSE for more details.
+"""
+
+
+class MapperError(Exception):
+    """Broken mapper configuration error."""
+
+    pass
```
Comment (sobolevn) [inline src/mappers/exceptions.py]: `pass` can be removed with no difference

File: src/mappers/_mapper.py (hunk 1 above)
Comment (proofit404) [reply src/mappers/_mapper.py]: We'll rewrite it into `MapperError` eventually.

File: src/mappers/exceptions.py (hunk 2 above)
Comment (proofit404) [reply src/mappers/exceptions.py]: Yes, but I'll keep it to be consistent with `dependencies` and `stories`.

**Comment by proofit404** [comment]
@sobolevn Thanks for the review!

File: tests/helpers/readers/annotations.py [hunk 3]
```diff
@@ -17,6 +17,22 @@ def load_users() -> List[user]:
     return load_users
 
 
+def get_load_user(mapper, user, user_id):
+    @mapper.reader
+    def load_user(primary_key: user_id) -> user:
```
Comment (sobolevn) [inline tests/helpers/readers/annotations.py]: This annotation does not look correct to me. Is it intended?

Comment (proofit404) [reply tests/helpers/readers/annotations.py]: Yes, it's part of the parametrized test, user id could be defined in one of the five examples of fixture modules.

**Comment by sobolevn** [comment]
Is https://github.com/wemake-services/django-test-migrations a good fit for this project?

**Comment by proofit404** [comment]
[django-test-migrations](https://github.com/wemake-services/django-test-migrations) is a great library! Thank you for creating it!

It's interesting how the end-user could use mappers together with the library. Or how the library could improve the development process of the mappers itself.

Can you elaborate more on what you suggest?

Best regards,
Artem.

**Comment by sobolevn** [comment]
Oh, I see. We only have migrations inside the test app.

**Comment by proofit404** [comment]
It could be good feet for our tutorials bookshelf example.

**Pull request merged**